src/fu-remote-list.c

```diff
@@ -658,6 +658,25 @@ fu_remote_list_load_metainfos(XbBuilder *builder, GError **error)
 	return TRUE;
 }
 
+gboolean
+fu_remote_list_toggle_test_remotes(FuRemoteList *self, gboolean enable, GError **error)
```

Comment:
something like fu_remote_list_set_testing_remote_enabled() would be better -- toggle changes true->false and false->true.